### ISSUE TITLE
Instructions for non-vim keybindings in README.md don't work

### ISSUE THREAD

**Issue by sogaiu** [mauricioszabo/atom-chlorine#35]
The keybinding instructions in README.md for "If you don't use vim bindings" includes the line:

```
'atom-text-editor.vim-mode-plus.normal-mode[data-grammar="source clojure"]':
```

That didn't work here.

However, changing the line to the following did:

```
'atom-text-editor[data-grammar="source clojure"]':
```

**Comment by sogaiu** [comment]
On a side note, ctrl-, appears to be bound bringing up preferences here, so changing that to something else (here ctrl-; worked), might be a better suggestion.

**Comment by mauricioszabo** [comment]
Thanks for catching it! Fixed on `master`.

**Comment by seancorfield** [comment]
I have those key bindings in place on both Mac and Windows because that's what ProtoREPL uses (and I used ProtoREPL before I used Chlorine). It does mean that `ctrl-,` no longer brings up preferences but that's fine by me as I use it a lot less than these bindings.

ProtoREPL added `ctrl-alt-,` as a prefix as well but I find that much harder to use on Windows.

Next time I'm on my (Windows) laptop, I'll double-check the behavior and come back and confirm.
